ttnn/cpp/ttnn/operations/moreh/moreh_group_norm_backward/device/input_grad/moreh_group_norm_backward_input_grad_factory.cpp

```diff
@@ -222,6 +216,9 @@ operation::ProgramWithCallbacks moreh_groupnorm_backward_input_grad_impl(
 
     const auto gamma_addr = gamma_has_value ? gamma.value().buffer()->address() : 0;
 
+    // std::cout << "output_grad_addr: " << output_grad_addr << std::endl;
+
+    // std::cout << "num_cores_to_be_used: " << num_cores_to_be_used << std::endl;
```

Comment:
Please remove the debugging code.
This debugging code is still there.

Reply:
Sorry for that, I must have forgotten about it while picking an old commit from Shaw.  I've fixed it now.